examples/chip-tool/commands/common/CommandInvoker.h

```diff
@@ -115,9 +124,10 @@ class CommandInvoker final : public ResponseReceiver<typename RequestType::Respo
         {
             return CHIP_ERROR_NO_MEMORY;
         }
-        ReturnErrorOnFailure(commandSender->SendCommandRequest(session.Value()));
 
+        ReturnErrorOnFailure(commandSender->SendGroupCommandRequest(session.Value()));
         commandSender.release();
```

Comment:
Should document that at this point `commandSender` and `this` will both be deleted already, so we should not be touching either one anymore.

Reply:
done